helper/DataGenerator.php

```diff
@@ -57,6 +59,55 @@ public static function generateItems($count = 100) {
         
         return $class;
     }
+
+    /**
+     * Generates $count of problematic multilanguage items. The secondary item language version will have
+     * modification time a second after the default
+     *
+     * @param int $count
+     * @return array
+     */
+    public static function generateMultilanguageItems($count = 100)
+    {
+        $ext = \common_ext_ExtensionsManager::singleton()->getExtensionById('taoDevTools');
+
+        $topClass = new \core_kernel_classes_Class(TaoOntology::ITEM_CLASS_URI);
+        $class = $topClass->createSubClass('Junk data generation (' . (new \DateTime())->format('Y-m-d H:i:s') . ')');
+
+        $sampleFile = $ext->getDir().'data/items/sampleItem.xml';
+
+        $generationData = [];
+        for ($i = 0; $i < $count; $i++) {
+
+            $report = ImportService::singleton()->importQTIFile($sampleFile, $class, false);
+            /** @var \core_kernel_classes_Resource $item */
+            $item = $report->getData();
+            $item->setLabel(NameGenerator::generateTitle());
+
+            $newLang = self::getRandomLanguage();
+            Service::singleton()->getDataItemByRdfItem($item, $newLang, false);
```

Comment:
Can you add a bit more random to the languages that exist for each item:
* have some items only in a non default language
* have some items only in the default language
* have some items in more than 2 languages

This way we get more variations and better test data. Do we also need to worry about assets in the images?